airmozilla/popcorn/views.py

```diff
@@ -1,5 +1,6 @@
-import requests
 from jsonview.decorators import json_view
+import requests
+from urlparse import urlparse
```

Comment:
`urlparse` is a standard python module and its import line should go first and separated (by a newline) from the third-party modules.